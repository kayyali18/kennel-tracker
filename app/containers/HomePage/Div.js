import styled from 'styled-components'

const Div = styled.div`
  display: inline-block;
  width: 250px;
  background: #e7b7d3;
  border: 2px dashed black;
  justify-content: space-between;
  padding: 10px;
  margin: 10px;
`
export default Div
