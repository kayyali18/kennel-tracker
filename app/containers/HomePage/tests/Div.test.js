import React from 'react'
import renderer from 'react-test-renderer'
import 'jest-styled-components'

import Div from '../Div'

describe('<Div />', () => {
  it('should match snapshot', () => {
    const renderedComponent = renderer.create(<Div />).toJSON()
    expect(renderedComponent).toMatchSnapshot()
  })

  it('should render a div', () => {
    const renderedComponent = renderer.create(<Div />).toJSON()
    expect(renderedComponent.type).toEqual('div')
  })
})
