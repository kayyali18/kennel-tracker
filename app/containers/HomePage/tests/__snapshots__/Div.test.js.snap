// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Div /> should match snapshot 1`] = `
.c0 {
  display: inline-block;
  width: 250px;
  background: #e7b7d3;
  border: 2px dashed black;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  padding: 10px;
  margin: 10px;
}

<div
  className="c0"
/>
`;
